tests/download/test_views.py

```diff
@@ -217,7 +217,7 @@ def test_get_document_metadata_document_store_error(client, store):
 
 
 def test_get_document_metadata_when_document_is_in_s3(client, store):
-    store.get_document_metadata.return_value = {'mimetype': 'text/plain'}
+    store.get_document_metadata.return_value = {'mimetype': 'text/plain', 'verify_email': False}
```

Comment:
Would this value ever be False, or would it be better to pass some kind of hash?

I know this doesn't change how the test works, but just so tests look realistic.

Reply:
This value is always only true or false in the return from `DocumentStore. get_document_metadata`. The hash is stored on the S3 object, but we don't ever send it out anywhere (and shouldn't - we only accept input and verify if it matches).

The `verify_email` here is saying 'email verification is required to access this object'